src/monitor/group_state_machine.c

```diff
@@ -329,7 +329,7 @@ AssignGoalState(AutoFailoverNode *pgAutoFailoverNode,
 						  pgAutoFailoverNode->nodeName,
 						  pgAutoFailoverNode->nodePort,
 						  pgAutoFailoverNode->pgsrSyncState,
-						  pgAutoFailoverNode->walDelta,
+						  pgAutoFailoverNode->currentLSN,
```

Comment:
On the monitor side, we get updated every 5 seconds. I reckon we should use `reportedLSN` here rather than _current_. Like the other `reported...` fields we already have.